tests/functional/api/test_simple.py

```diff
@@ -24,6 +24,8 @@ def test_simple_api_html(webtest):
 
 def test_simple_api_detail(webtest):
     project = ProjectFactory.create()
+    ReleaseFactory.create(version="1.0.0", project=project)
+    ReleaseFactory.create(version="2.0.0", project=project)
```

Comment:
totally fine, could also be:
```suggestion
    ReleaseFactory.create_batch(2, project=project)
```

Reply:
Does this ensure that the versions are unique? It's unclear from https://factoryboy.readthedocs.io/en/stable/reference.html#factory.create_batch

Reply:
We control the `version` with a sequence already to ensure uniqe: https://github.com/pypi/warehouse/blob/ca2781e6f4b947a9739fe05ac33315fcbac3075d/tests/common/db/packaging.py#L93